### ISSUE TITLE
bug: `--git-repos` shows Git status for `.git` directory

### ISSUE THREAD

**Issue by injust** [eza-community/eza#1085]
Running `eza --all --long --git-repos` _inside_ a Git repo:

![image](https://github.com/user-attachments/assets/2b866b14-9ff2-476f-bcc9-dbfbc02825a6)

It shows the "status" of the `.git` directory. This should be omitted -- the status should only be shown for the _root_ directory of a Git repo.

---

eza version: 0.19.0 [+git]
Shell: fish v3.7.1
Terminal: Ghostty
OS: macOS Sonoma 14.6 (Intel)

**Comment by eddyekofo94** [comment]
I am having the same problem on WSL

**Comment by cafkafk** [comment]
I don't think this is a major issue? Normally, you'll only see that with `-aa` anyways, I don't see the harm in having it for `.git`

**Comment by injust** [comment]
> Normally, you'll only see that with `-aa` anyways, I don't see the harm in having it for `.git`

Just `-a` (not `-aa`), which is pretty common.

The problem is that eza uses 2 extra columns to display the "status" of the `.git` directory, when run in the root directory of a Git repo.

<img width="581" height="524" alt="Image" src="https://github.com/user-attachments/assets/8fce4a78-93d6-4b59-afa6-f44c18514a26" />

I have most of my repos in a `~/code/` directory, so eza using 2 extra columns to show Git info there is what I want. But it's a waste of space when it happens in the root of every Git repo.
